cpp/oneapi/dal/table/detail/table_builder.hpp

```diff
@@ -191,7 +191,7 @@ class ONEDAL_EXPORT homogen_table_builder : public table_builder {
                     std::int64_t column_count,
                     const sycl::vector_class<sycl::event>& dependencies = {}) {
         auto& impl = get_impl();
-        detail::wait_and_throw(dependencies);
+        sycl::event::wait_and_throw(dependencies);
```

Comment:
Here we use `wait_and_throw`, but in memory copy routines `event::wait`. Why behavior is different?

Reply:
Changed to use `wait_and_throw` in memory routines